bin/sdk/gutenberg.js

```diff
@@ -95,6 +97,15 @@ exports.config = ( { argv: { inputDir, outputDir }, getBaseConfig } ) => {
 						betaBlocks: presetBetaBlocks,
 					},
 				} ),
+			new webpack.BannerPlugin( {
+				banner: [
+					new Date().toUTCString(),
+					`Commit: ${ String(
+						execSync( `git -C ${ path.join( inputDir, DIRECTORY_DEPTH ) } rev-parse HEAD` )
```

Comment:
I'd grab the `commitSha` as a constant, run `_.compact` over the whole array, and set this bit as
```suggestion
						commitSha && `Built from commit: ${ commitSha }`
```
What happens with the execSync if you're not in a repo? Is it something to wrap in a `try/catch`?

Reply:
With different possible `inputDir` paths (inside Calypso in different places or outside Calypso folder) I couldn't reliably check.

Tried if I could do some trickery with [`path.relative()`](https://nodejs.org/docs/latest/api/path.html#path_path_relative_from_to) but I don't have time to figure it out now.

So now I'm including commit sha only if we're building from Calypso repository, since that's easy to check by comparing `calypsoRoot` and `inputDir`.